plugins/modules/terraform.py

```diff
@@ -13,8 +13,15 @@
 module: terraform
 short_description: Manages a Terraform deployment (and plans)
 description:
-     - Provides support for deploying resources with Terraform and pulling
-       resource information back into Ansible.
+    - Provides support for deploying resources with Terraform and pulling
+      resource information back into Ansible.
+extends_documentation_fragment:
+    - community.general.attributes
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
```

Comment:
Rest of the file is using 2 spaces
```suggestion
  - Provides support for deploying resources with Terraform and pulling
    resource information back into Ansible.
extends_documentation_fragment:
  - community.general.attributes
attributes:
  check_mode:
    support: full
  diff_mode:
    support: none
```